src/credentials/FabricTable.h

```diff
@@ -123,7 +123,30 @@ class DLL_EXPORT FabricInfo
         }
         return mOperationalKey;
     }
-    CHIP_ERROR SetOperationalKeypair(const Crypto::P256Keypair * keyPair);
+
```

Comment:
`GetOperationalKey` creates an operational key if one isn't present, but it should likely consult with `mHasExternallyOwnedOperationalKey` to make that determination. There is a possibility that can be 'true' BUT you have yet to actually fill in the externally provided key (i.e the time window between fabric infos being loaded from storage, to when the Controller is initialized).

See comment above.